subprojects/platform-jvm/src/main/java/org/gradle/api/java/archives/Manifest.java

```diff
@@ -101,4 +103,17 @@
      * @return this
      */
     Manifest from(Object mergePath, Closure<?> closure);
+
+    /**
+     * Specifies other manifests to be merged into this manifest. A merge path is interpreted as described in
+     * {@link #from(Object...)}.
+     *
+     * The merge is not happening instantaneously. It happens either before writing or when {@link #getEffectiveManifest()}
+     * is called.
+     *
+     * @return this
+     * @since 5.0
+     */
+    @Incubating
```

Comment:
@oehme, I'm tempted to not mark this as `@Incubating` and accept the change in the compatibility check. Would that be an acceptable policy to apply to such overloads for 5.0?